File: java/src/main/java/com/gushenge/atools/util/ALog.java
package com.gushenge.atools.util;

import android.util.Log;

public class ALog {
    public static void v(String TAG,Object message){
        Log.v(TAG,message.toString());
    }
    public static void d(String TAG,Object message){
        Log.d(TAG,message.toString());
    }
    public static void w(String TAG,Object message){
        Log.w(TAG,message.toString());
    }
    public static void i(String TAG,Object message){
        Log.i(TAG,message.toString());
    }
    public static void e(String TAG,Object message){
        Log.e(TAG,message.toString());
    }
}

File: kotlin/src/main/java/com/gushenge/atools/util/ALog.kt
package com.gushenge.atools.util

import android.util.Log

@Deprecated("",level = DeprecationLevel.WARNING)
object ALog {
    fun v(TAG:String,message:Any){
        Log.v(TAG,message.toString())
    }
    fun d(TAG:String,message:Any){
        Log.d(TAG,message.toString())
    }
    fun i(TAG:String,message:Any){
        Log.i(TAG,message.toString())
    }
    fun w(TAG:String,message:Any){
        Log.w(TAG,message.toString())
    }
    fun e(TAG:String,message:Any){
        Log.e(TAG,message.toString())
    }
}